### PR TITLE
Feature 15 set up master theme

### PR DESCRIPTION
**Pull request by JuusoMRusanen**
Request review

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -3,6 +3,8 @@ import DashboardScreen from "./components/screens/dashboard/dashboard-screen";
 import ErrorScreen from "./components/screens/error-screen";
 import AuthenticationProvider from "./components/providers/authentication-provider";
 import ErrorHandler from "./components/contexts/error-handler";
+import { ThemeProvider } from "@emotion/react";
```
Comment (Jdallos) [inline src/App.tsx:6]: This looks like a different import from what is used in OSS, is that deliberate? I'm not familiar with the details for setting up a theme, so if it works and you're happy with it then so am I!

**Review by Jdallos — APPROVED** [review]
Approved with a comment

File: src/App.tsx (hunk 1 above)
Comment (JuusoMRusanen) [reply src/App.tsx:6]: According to the docs, createTheme is imported from "@mui/material". Thanks for pointing that out!

**Review by petri-nykanen — APPROVED** [review]
Approved

**Pull request merged**